revolv/static/js/base/project.js

```diff
@@ -0,0 +1,68 @@
+$(document).ready(function () {
+	drawPercentagePartialCircle()	
+
```

Comment:
wait why can this be deleted?
doesn't it draw the circle after the document is ready?